test/spec/core/image.spec.js

```diff
@@ -32,12 +32,11 @@ describe("Image", function() {
   });
 
   it("should generate the URL for a torque layer", function(done) {
-
     var vizjson = "http://documentation.carto.com/api/v2/viz/3ec995a8-b6ae-11e4-849e-0e4fddd5de28/viz.json"
 
     var image = cartodb.Image(vizjson);
 
-    var regexp = new RegExp("http://a.ashbu.cartocdn.com/documentation/api/v1/map/static/bbox/(.*?)/-138\.6474609375,27\.761329874505233,-83\.408203125,51\.26191485308451/320/240\.pn");
+    var regexp = new RegExp("http://a.gusc.cartocdn.com/documentation/api/v1/map/static/bbox/(.*?)/-138\.6474609375,27\.761329874505233,-83\.408203125,51\.26191485308451/320/240\.pn");
```

Comment:
https here too?

Reply:
Nope! Endpoint returns it without HTTPS so...